cdap-docs/faqs/source/cdap.rst

```diff
@@ -87,6 +64,20 @@ If you have followed :ref:`the installation instructions <install>`, and CDAP ei
   installed the packages and started the services).
 
 
+Using YARN with the LinuxContainerExecutor.
```

Comment:
Revised FAQs to match this order and added rubrics to help divide up the questions.

See [FAQs: CDAP](http://builds.cask.co/artifact/CDAP-DOB48/shared/build-6/Docs-HTML/3.2.1/en/faqs/cdap.html)